source/middleware/LocalizationMiddleware.ts

```diff
@@ -0,0 +1,21 @@
+import type { Request, Response, NextFunction } from 'express';
+import en from '../localization/lang_en.js';
+import lv from '../localization/lang_lv.js';
+import type locale from '../localization/localization.js';
+
+const localizations: Record<string, locale> = {
+	en,
+	lv
+};
+
+export function localization (req: Request, res: Response, next: NextFunction): void {
+	const lang = req.cookies.lang;
+	console.log(localizations);
```

Comment:
Did you mean to leave this `console.log` in?

Reply:
No, i did not
I'll remove it